packages/server/email/teamLimitsEmailCreator.tsx

```diff
@@ -0,0 +1,107 @@
+import LockedEmail from 'parabol-client/modules/email/components/LimitsEmails/LockedEmail'
+import SevenDayWarningEmail from 'parabol-client/modules/email/components/LimitsEmails/SevenDayWarningEmail'
+import ThirtyDayWarningEmail from 'parabol-client/modules/email/components/LimitsEmails/ThirtyDayWarningEmail'
+import {PALETTE} from 'parabol-client/styles/paletteV3'
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+import {Threshold} from '../../client/types/constEnums'
+import makeAppURL from '../../client/utils/makeAppURL'
+import appOrigin from '../appOrigin'
+import {TeamLimitsEmailType} from '../billing/helpers/sendTeamsLimitEmail'
+import {analytics} from '../utils/analytics/analytics'
+import emailTemplate from './emailTemplate'
+
+const textOnlySummary = (props: Props) => {
+  const {preferredName, orgId, emailType, orgName} = props
+  const billingURL = makeAppURL(appOrigin, `/me/organizations/${orgId}/billing`, {
+    searchParams: {
+      utm_source: 'notification email',
+      utm_medium: 'email',
+      utm_campaign: 'locked-email'
+    }
+  })
+
+  const emailTextLookup = {
+    thirtyDayWarning: `Hi ${preferredName} 👋
+
+    This is a friendly note to let you know that ${orgName} has officially reached ${Threshold.MAX_PERSONAL_TIER_TEAMS} active teams on Parabol - congrats! We love to see organizations finding value in Parabol and improving their teams in the process.
+
+    As a reminder: Parabol's Starter Plan has a limit of two teams. Please upgrade your account to continue using Parabol with all of your teams: ${billingURL}
+
+    If you aren’t ready to upgrade, then in 30 days your account will revert to the two teams allowed on the Starter Plan. Feel free to contact us with any questions - we’re here to help!
+
+    Parabol Team
+  `,
+    sevenDayWarning: `Hi ${preferredName} 👋
+
+    This is a follow-up notification to remind you that ${orgName}'s Parabol account is at risk of deactivation because you’ve exceeded the two team limit on your Starter Plan.
+
+    Once your account is deactivated, you will lose access to your teams and won’t be able to run Retrospective, Sprint Poker or Standup meetings with Parabol.
+
+    You'll need to upgrade your account within the next 7 days to avoid losing access to your agile meetings: ${billingURL}
```

Comment:
Should we use a variable instead of '7 days' ?

Reply:
Sounds good 👍